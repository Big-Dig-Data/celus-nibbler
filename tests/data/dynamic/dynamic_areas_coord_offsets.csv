MyOrg


Report
MyMetric

2020,
Jan,Feb,Mar
1,2,3

2021,
Jan,Feb,Mar
2,3,4


2022,
Jan,Feb,Mar
3,4,5
