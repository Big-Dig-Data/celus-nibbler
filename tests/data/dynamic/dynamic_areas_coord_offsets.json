{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "dynamic_areas_coord_offsets",
  "data_format": {
    "name": "simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "possible_row_offsets": [0, 1, 2, 3],
  "heuristics": {
    "conds": [
      {
        "pattern": "Report",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "max_areas_generated": null,
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 0
                    },
                    "direction": "right"
                },
                "composed": {
                    "year": {
                        "source": {
                          "row": 0,
                          "col": 0
                        },
                        "date_pattern": "%Y",
                        "role": "date"
                    },
                    "month": {
                        "source": {
                            "coord": {
                              "row": 1,
                              "col": 0
                            },
                            "direction": "right"
                        },
                        "date_pattern": "%b",
                        "role": "date"
                    }
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 2,
            "col": 0
          },
          "direction": "right"
        },
        "condition": {
          "conds": [
            {
              "kind": "is_date",
              "date_format": "%Y",
              "coord": {
                "row": 0,
                "col": 0
              }
            }
          ],
          "kind": "and"
        }
      },
      "metrics": {
        "source": {
            "row": 1,
            "col": 0,
            "row_relative_to": "parser"
        }
      },
      "organizations": {
        "source": {
            "row": 0,
            "col": 0,
            "row_relative_to": "start"
        }
      },
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.generic"
    }
  ]
}
