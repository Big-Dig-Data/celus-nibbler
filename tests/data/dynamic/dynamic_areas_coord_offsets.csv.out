"2020-01-01","2020-01-31","MyOrg","","","","","","","","MyMetric","1"
"2020-02-01","2020-02-29","MyOrg","","","","","","","","MyMetric","2"
"2020-03-01","2020-03-31","MyOrg","","","","","","","","MyMetric","3"
"2021-01-01","2021-01-31","MyOrg","","","","","","","","MyMetric","2"
"2021-02-01","2021-02-28","MyOrg","","","","","","","","MyMetric","3"
"2021-03-01","2021-03-31","MyOrg","","","","","","","","MyMetric","4"
"2022-01-01","2022-01-31","MyOrg","","","","","","","","MyMetric","3"
"2022-02-01","2022-02-28","MyOrg","","","","","","","","MyMetric","4"
"2022-03-01","2022-03-31","MyOrg","","","","","","","","MyMetric","5"
